includes/admin.form.inc

```diff
@@ -18,6 +18,7 @@
  */
 function islandora_paged_content_admin_settings_form(array $form, array &$form_state) {
   form_load_include($form_state, 'inc', 'islandora_paged_content', 'includes/admin.form');
+  module_load_include('inc','islandora','/includes/utilities');
```

Comment:
Not sure, but should we not use `form_load_include()` inside forms (as per [documentation](https://api.drupal.org/api/drupal/includes!form.inc/function/form_load_include/7.x), this makes sure the file is loaded when the form gets processed also when the form was cached)?

Also, even though it might not make a difference, but is there a reason for the leading slash in the `$name` parameter?

Finally, there should be spaces after the commas (TravisCI - the other needed fix is an empty line before `* DEPRECATED: ...` in the function comment of `islandora_paged_content_admin_settings_form_executable_available_message()`, [L180](https://github.com/Islandora/islandora_paged_content/pull/152/files#diff-0cdd19abe14e51fe9e7f061ec72c86d3R180)).

Reply:
Good call on form_load_include! I'll go clean that up.